test/src/test/java/org/corfudb/integration/WorkflowIT.java

```diff
@@ -112,6 +112,19 @@ public void AddNodeIT() throws Exception {
         final int clusterSizeN3 = 3;
         assertThat(n1Rt.getLayoutView().getLayout().getAllServers().size()).isEqualTo(clusterSizeN3);
 
+        // Remove node 2
+        n1Rt.getManagementView().removeNode(getConnectionString(n2Port), workflowNumRetry,
+                timeout, pollPeriod);
+        n1Rt.invalidateLayout();
+        assertThat(n1Rt.getLayoutView().getLayout().getAllServers().size()).isEqualTo(clusterSizeN2);
```

Comment:
I think you should also assert that all the servers (logunit, sequencer, layout) are as expected in the new layout.

Reply:
Done.